DESCRIPTION

```diff
@@ -1,13 +1,13 @@
 Package: gpdd
-Title: What the Package Does (One Line, Title Case)
+Title: Download datasets from the Global Population Dynamics Database.
 Version: 0.0.0.9000
 Authors@R: 
     person(given = "First",
            family = "Last",
            role = c("aut", "cre"),
            email = "first.last@example.com")
-Description: What the package does (one paragraph).
+Description: Download data from GPDD (https://knb.ecoinformatics.org/view/doi:10.5063/F1BZ63Z8).
```

Comment:
Longer description please.  This should be easy for anyone to understand.  You may want to use text directly from the abstract for the GPDD itself.  Also, the URL should use the "canonical" format: https://doi.org/10.5063/F1BZ63Z8.

Reply:
I just enriched the description to a paragraph. Please feel free to take a look, and let me know if further modifications are needed.